fmdtools/analyze/graph.py

```diff
@@ -1032,96 +1159,182 @@ def data_average(data):
             closest_pt = None
         return closest_pt
+
     def on_button_press(self, event):
-        """Determines what to do when a button is pressed"""
+        """
+        Determines what to do when a button is pressed
+
+        Parameters
+        ----------
+        event
+
+        Returns
+        -------
+        Returns nothing (intended)
+
+        """
+        """"""
         if event.inaxes is None:
             return
-        if event.inaxes==self.bax:
+        if event.inaxes == self.bax:
             self.print_pos()
             return
         if event.button != 1:
             return
         self._clicked_node = self.get_closest_point(event)
+
     def on_button_release(self, event):
-        """Determines what to do when the mouse is released"""
+        """
+        Determines what to do when the mouse is released
+
+        Parameters
+        ----------
+        event
+
+        Returns
+        -------
+        Returns nothing (intended)
+        """
         if event.button != 1:
             return
         self._clicked_node = None
         self.ax.clear()
         self.refresh_plot()
+
     def on_mouse_move(self, event):
-        """Changes the node position when the user drags it"""
+        """
+        Changes the node position when the user drags it
+        Parameters
+        ----------
+        event
+
+        Returns
+        -------
+        Returns nothing (intended)
+        """
         if not self.showverts:
             return
         if event.inaxes is None:
             return
         if event.button != 1:
             return
         x, y = event.xdata, event.ydata
-        if self._clicked_node: self.g_obj.pos[self._clicked_node]=[x,y]
+        if self._clicked_node:
+            self.g_obj.pos[self._clicked_node] = [x, y]
+
     def refresh_plot(self):
-        """Refreshes the plot with the new positions."""
-        self.g_obj.pos = {pt:np.round(loc,2) for pt, loc in self.g_obj.pos.items()}
+        """
+        Refreshes the plot with the new positions.
+
+        Returns
+        -------
+
+        """
+        self.g_obj.pos = {pt: np.round(loc, 2) for pt, loc in self.g_obj.pos.items()}
         self.g_obj.draw(fig=self.fig, ax=self.ax, withlegend=False, **self.kwargs)
-        self.ax.set_xlim(-1,1)
-        self.ax.set_ylim(-1,1)
-        limits=plt.axis('on')
+        self.ax.set_xlim(-1, 1)
+        self.ax.set_ylim(-1, 1)
+        limits = plt.axis('on')  # TODO : Looks like limits is not used and might be removed from the code.
         self.ax.tick_params(left=True, bottom=True, labelleft=True, labelbottom=True)
         self.ax.set_aspect('equal')
         self.ax.grid(True, which='both')
         self.ax.set_title('Drag nodes to change their positions')
         self.t+=1
         plt.pause(0.001)
+
     def print_pos(self):
-        print({k:list(v) for k,v in self.g_obj.pos.items()})
+        """
+        TODO : Needs documentation
+        Returns
```

Comment:
Prints the current node positions in the graph so they can be viewed (and copied) from the console.

Reply:
Done